examples/mlx_metal_kernel_opt/qwen3_benchmark_suite.py

```
@@ -0,0 +1,978 @@
+                "-m",
+                "mlx_lm.generate",
+                "--model",
+                self.model_path,
+                "--prompt",
+                config.prompt,
+                "--max-tokens",
+                str(config.max_tokens),
+            ]
+
+            # Clear MLX cache before starting
+            print(f"🧹 Clearing MLX cache...")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)